src/client/react.js

```diff
@@ -65,6 +73,7 @@ export function Client({ game, numPlayers, board, multiplayer, debug }) {
         multiplayer,
         gameID: props.gameID,
         playerID: props.playerID,
+        enhancer: enhancer,
```

Comment:
Can just omit the value (similar to some of the fields above).

Reply:
oh totally, sure

Reply:
@nicolodavis fixed